README.md

```diff
@@ -1494,7 +1494,35 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
-* <a id='limit-vertical-whitespace'></a>(<a href='#limit-vertical-whitespace'>link</a>) **Limit empty vertical whitespace to one line.** Favor the following formatting guidelines over whitespace of varying heights to divide files into logical groupings. [![SwiftLint: vertical_whitespace](https://img.shields.io/badge/SwiftLint-vertical__whitespace-007A87.svg)](https://github.com/realm/SwiftLint/blob/master/Rules.md#vertical-whitespace)
```

Comment:
One of the reasons we've named these links is so that each rule has a permalink. When we created the style guide we were told this was a good idea by the folk working on the JS style guide. With this PR, the https://github.com/airbnb/swift#limit-vertical-whitespace permalink breaks. Is there any way to continue to have https://github.com/airbnb/swift#limit-vertical-whitespace bring us to this rule?

Reply:
Sure, I added back an anchor for `#limit-vertical-whitespace` (`<a id='limit-vertical-whitespace'></a>`)